kolibri/core/assets/src/utils/browserInfo.js

```diff
@@ -64,6 +64,13 @@ export const os = {
   patch: osVersion[2],
 };
 
+// Device info
+export const device = {
+  type: info.device.type || 'desktop',
```

Comment:
Why are we defaulting to 'desktop'?

Reply:
The ua-parser library gives undefined for desktop. As according to them `desktop` is not an device type.

Reply:
https://github.com/faisalman/ua-parser-js/issues/182